tests/.dockerignore

```diff
@@ -0,0 +1,18 @@
+clueso-tests/
```

Comment:
Not exactly pertinent to this PR but at some point "clueso-tests" should probably be updated to something more generic like "metadata-search-tests"

Reply:
Agree, though indeed not something for this PR :)